htdocs/install/makedata.php

```diff
@@ -85,11 +85,6 @@ function make_data(&$dbm, &$cm, $adminname, $adminlogin_name, $adminpass, $admin
 		. ", (0,".$gruops['XOOPS_GROUP_ADMIN'].",1,1,'content_admin')"
 		. ", (0,".$gruops['XOOPS_GROUP_ADMIN'].",1,1,'use_wysiwygeditor')");
 
-	// data for table 'banner'
-	$dbm->insert("banner", " (bid, cid, imptotal, impmade, clicks, imageurl, clickurl, date, htmlcode) VALUES (1, 1, 0, 1, 0, '".XOOPS_URL."/images/banners/impresscms_banner.gif', '"._INSTALL_LOCAL_SITE."', 1008813250, '')"
```

Comment:
I think you should remove related files to. Doesn't make sense to have banner folder that is not used anymore.

Reply:
Normally the content of the banner folder, as well as the folder, is removed as well (update.php line 117 - 121).